README.md

```diff
@@ -26,14 +26,28 @@ The firmware for the Original Prusa i3 printers is proudly based on [Marlin 1.0.
 
 1. Clone this repository and checkout the correct branch for your desired release version.
 
-2. Set your printer model. 
+1. Set your printer model. 
```

Comment:
numbering got broken

Reply:
I removed numbering on purpose, so it is numbered automatically, and doesn't need to be renumbered once another point is added.